### ISSUE TITLE
Clipboard buttons don't work

### ISSUE THREAD

**Issue by ddrx1984** [pglombardo/PasswordPusher#1667]
Hello, I hope you can understand me, I use Google Translate.
I set up the password pusher in Docker and everything worked great.
But unfortunately the "clipboard buttons" don't work, the buttons are like dead without any function and nothing is put on the "clipboard".
copied.
My command for Docker is like this:
docker run --name pwpush1 -d -p "5100:5100" \
-e PWP__BRAND__LIGHT_LOGO=/logos/mylogo.png \
-e PWP__DEFAULT_LOCALE=de \
-v /var/lib/pwpush/logos:/opt/PasswordPusher/public/logos \
pglombardo/pwpush:release

Would it be possible to fix the “clipboard copy”?
I have version v1.34.1

also look at the screenshot

![button1](https://github.com/pglombardo/PasswordPusher/assets/152288212/724a6c03-3b6c-4f5a-a182-d6481d00d1dc)

![Button2](https://github.com/pglombardo/PasswordPusher/assets/152288212/60fc2605-afa1-4c8b-a622-d8aa8b109806)

Thank you very much

**Comment by pglombardo** [comment]
Hi @ddrx1984 - the copy buttons won't work unless you are hosting with SSL.   #1020 has a bit more information.

Are you hosting in plain HTTP?

**Comment by ddrx1984** [comment]
good morning @pglombardo,
version v1.34.3 still has the error.
Can you tell me how long the error would be fixed?
Thank you

**Comment by pglombardo** [comment]
Hi @ddrx1984

> Can you tell me how long the error would be fixed?

No idea currently.  This is a browser security based issue.  I might be able to find a work around but I'm not sure when as I often am juggling between this project and my day job.  I'll update this issue once I have an update.  Unfortunately I don't have an ETA.

> hi, would the errors with the clipboard be fixed in the latest version?

No

**Comment by pglombardo** [comment]
Hi @ddrx1984 - just as a late response:

> But I'm wondering why the "clipboard" button works on the website: https://pwpush.com/?

pwpush.com com is hosted via `https`.  The bug exists on `http` sites.

**Comment by ddrx1984** [comment]
> Hi @ddrx1984 - just as a late response:
> 
> 
> 
> > But I'm wondering why the "clipboard" button works on the website: https://pwpush.com/?
> 
> 
> 
> pwpush.com com is hosted via `https`.  The bug exists on `http` sites.

Yes, I know that, and that's why I reported it to you. For me it runs locally without https. it would be nice if you could fix this bug. Thanks

**Comment by pglombardo** [comment]
Hi all - I just tested this again and copy to clipboard is now working on HTTP.  Tested on Chrome and Safari.

I'll close this issue out but if anything remains we can re-open if needed.

Thanks all for reporting and feedback!
